namespaces/context_test.go

```diff
@@ -69,3 +71,24 @@ func TestNamespaceFromEnv(t *testing.T) {
 		t.Fatalf("unexpected namespace: %q != %q", namespace, expected)
 	}
 }
+
+func TestNamespaceRequired(t *testing.T) {
+	ctx := context.Background()
+	namespace, ok := Namespace(ctx)
+	if ok {
+		t.Fatal("namespace should not be present")
+	}
+
+	if namespace != "" {
+		t.Fatalf("namespace should not be defined: got %q", namespace)
+	}
```

Comment:
I don't understand the value of the added test having an exact copy of the test on line 26 (`TestContext`)? The rest of this I understand as it tests `NamespaceRequired` explicitly versus just checking that a namespace exists once set.

Reply:
OK, I will close it